WalletWasabi/Crypto/StrobeProtocol/StrobeHasher.cs

```diff
@@ -23,9 +25,20 @@ public static StrobeHasher Create(string domain)
 		public StrobeHasher Append(string fieldName, IBitcoinSerializable serializable)
 			=> Append(fieldName, serializable.ToBytes());
 
+		public StrobeHasher Append(string fieldName, TimeSpan time)
+			=> Append(fieldName, time.Ticks);
+
 		public StrobeHasher Append(string fieldName, Money money)
 			=> Append(fieldName, money.Satoshi);
 
+		public StrobeHasher Append(string fieldName, MoneyRange range)
+			=> Append(fieldName + "-min", range.Min).Append(fieldName + "-max", range.Max);
+
+		public StrobeHasher Append(string fieldName, IEnumerable<ScriptType> scriptTypes)
```

Comment:
```suggestion
		public StrobeHasher Append(string fieldName, ImmutableSortedSet<ScriptType> scriptTypes)
```

What do you think about being more specific about the concrete type here, since it implies a canonical ordering, not just a given ordering? This is strictly a readability improvement, `IEnumerable` is just as correct, but I think it will always be an immutable sorted set and this will save someone a brain cycle or two of worrying about where the order is specified the next time they read it.